### PR TITLE
Added ducks.party instance, set hostname for redis container

### PR DESCRIPTION
**Pull request by nyuuzyou**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ services:
   priviblur-redis:
     image: redis:6.2-alpine
     container_name: priviblur-redis
+    hostname: redis
```
Comment (TheFrenchGhosty) [inline docker-compose.yml:17]: Don't do that.

File: config.example.toml [hunk 2]
```diff
@@ -22,7 +22,8 @@
 # Ignore to disable the cache
 #
 # [cache]
-    # url =
+    # If you are using Docker Compose, set the URL to the Redis container like this:
+    # url = "redis://redis:6379"
```
Comment (TheFrenchGhosty) [inline config.example.toml:26]:
```suggestion
    # url = "redis://priviblur-redis:6379"
```

**Review by TheFrenchGhosty — CHANGES_REQUESTED** [review]

**Review by syeopite — CHANGES_REQUESTED** [review]
Can you remove the docker and config changes? I'll be merging this alongside #37 which implements the changes here and then some.

**Comment by syeopite** [comment]
@nyuuzyou Your instance is more than a month out of date! Please update it or I'll have to remove it from the instance list.

**Pull request merged**